Valet.xcodeproj/project.pbxproj

```diff
@@ -497,6 +639,106 @@
 /* End PBXVariantGroup section */
 
 /* Begin XCBuildConfiguration section */
+		26E682811BA8B3F900EFF4EA /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
+				CURRENT_PROJECT_VERSION = 1;
```

Comment:
@eliperkins any idea how this value is used / if it is used by Carthage?

My guess is "no" given [Apple's documentation on how to use it](https://developer.apple.com/library/ios/qa/qa1827/_index.html). But it's a little odd that the CURRENT_PROJECT_VERSION doesn't match the actual framework version.